static/styles/zulip.scss

```diff
@@ -390,6 +390,9 @@ li,
     left: -5px;
     margin-top: -24px;
     display: block;
+    z-index: 2;
+    /* added z-index to account for decreased
+    opacity of textarea when it is readonly */
```

Comment:
The comment should be above the line, and I would phrase it differently:
* "added" isn't appropriate for a code comment, since it should be explaining what the code does or why, not a change.

I would write something like "The partial opacity of this read-only textarea means that it we need this z-index to ensure the copy-message icon is clickable."